### DISCUSSION TITLE
using checkpointer to  initialize another model

### DISCUSSION THREAD

**Discussion by ammtD** [CliMA/Oceananigans.jl#2590] [Computational science!]
How does one use the output from one simulation, stored in a checkpointer, to initialize fields and grid of another simulation?
I have tried to load model.velocities  and tracer from the stored checkpointer file
```
spinup = jldopen(spinup.jld2)
grid = spinup["grid"]
model = NonhydrostaticModel(; grid, ... )
```     (other items of model redefined)
```
u_ini = spinup["u/data"]
v_ini = spinup["v/data"]
w_ini = spinup["w/data"]
b_ini = spinup["b/data"]
```
 but then on trying to` set!(model, u=u_ini, v=v_ini, w=w_ini, b=b_ini)`
I get the error message
 _DimensionMismatch("array could not be broadcast to match destination")_
...
the fields u_ini etc. contain the halo (3x3x3) halo's, which I guess may be the problem?

**Comment by glwagner** [comment]
Check out the documentation here: https://clima.github.io/OceananigansDocumentation/stable/model_setup/checkpointing/
